### PR TITLE
internal: add comments and map [32] byte to string

### PR DESCRIPTION
**Pull request by markya0616**

### CODE REVIEW

**Review by alanchchen — APPROVED** [review]

File: internal/impl/contract.go [hunk 1]
```diff
@@ -47,7 +48,8 @@ func (c *Contract) IsServerInterface(name string) bool {
 	return false
 }
 
-var ContractTemplate string = `package {{ .Package }};
+var ContractTemplate string = `// This file is automatically generated by grpc-contract, please do not modify.
```
Comment (tailingchen) [inline internal/impl/contract.go]:
```
// Automatically generated by grpc-contract. DO NOT EDIT!
// source:
```
add source file path

File: internal/impl/utils.go [hunk 2]
```diff
@@ -27,7 +27,8 @@ type Utils struct {
 	Package string
 }
 
-var UtilsTemplate string = `package {{ .Package }};
+var UtilsTemplate string = ` // This file is automatically generated by grpc-contract, please do not modify.
```
Comment (tailingchen) [inline internal/impl/utils.go]:
```
// Automatically generated by grpc-contract. DO NOT EDIT!
// source:
```
add source file path

File: internal/impl/contract.go (hunk 1 above)
Comment (markya0616) [reply internal/impl/contract.go]: OK.

File: internal/impl/utils.go (hunk 2 above)
Comment (markya0616) [reply internal/impl/utils.go]: OK

**Review by tailingchen — APPROVED** [review]

**Pull request merged**